testpilot/frontend/static-src/styles/_layout.scss

```diff
@@ -85,3 +85,13 @@
 .overflow-hidden {
   overflow: hidden;
 }
+
+.show-on-small {
+  @include respond-to('not-small') {
+    display: none;
+  }
+
+  @include respond-to('small') {
```

Comment:
It might make more sense as a mixin, now that I think about it. Single param, default `block`, can be overriden when appropriate.